src/panels/config/cloud/google-assistant/cloud-google-assistant.ts

```diff
@@ -174,17 +179,51 @@ class CloudGoogleAssistant extends LitElement {
                 secondary-line
                 @click=${this._showMoreInfo}
               >
-                ${entity.entity_id in this.hass.entities
-                  ? html`<button
-                      class="link"
-                      .entityId=${entity.entity_id}
-                      @click=${this._openAliasesSettings}
-                    >
-                      ${this.hass.localize(
-                        "ui.panel.config.cloud.google.manage_aliases"
-                      )}
-                    </button>`
-                  : ""}
+                ${aliases
+                  ? html`
+                      <span>
+                        ${aliases.length > 0
+                          ? [...aliases]
+                              .sort((a, b) =>
+                                stringCompare(a, b, this.hass.locale.language)
+                              )
+                              .join(", ")
+                          : this.hass.localize(
+                              "ui.panel.config.cloud.google.no_aliases"
+                            )}
+                      </span>
+                      <br />
+                      <button
+                        class="link"
```

Comment:
Can we just use `mwc-button` here?  Something styled link should take me somewhere else, not open a dialog.  Seems inconsistent with the rest of the UI (and arguably a WCAG failure).

Reply:
> Reaction on initial screenshot: when an alias / no alias set label is clickable, let's make sure it's underlined to indicate that.

Updated design
![CleanShot 2023-01-05 at 15 38 01](https://user-images.githubusercontent.com/5878303/210805420-a540a536-8175-400f-9104-72d733f54f2b.png)

Reply:
Yes I agree it should be a button. But we can not use `mwc-button` because it would be to important in the UI. The design/UX is not perfect because we wanted to provide alias to this page without impact to much the current design. 

However, we plan to replace this page with a data table very soon to be consistent with other panels. "Manage aliases" action will move into a overflow menu on the left like we have for automations, scripts, etc...

Reply:
In light of the redesign plan, I removed my request for changes, but please note that use of the `buttonLinkStyle` in general has a11y issues.  It looks like a link, but maps to a button role in the accessibility tree, leading to issues such as:

- Someone using voice software like Dragon Naturally Speaking might give the commend to "click the manage aliases link", which doesn't work because there is no such link.
- Documentation or a user forum might help a screen reader user by saying click the link to manage aliases.  They then use screen reader shortcuts to navigate through links, and get confused when they don't find it.

Aside from that, there's the potential implication to users with cognitive disabilities which are always more difficult to define concretely.

Reply:
I noticed some place where `buttonLinkStyle` is used. I agree we should avoir using that style.